### PR TITLE
Feature/leuven chapter3

### PR DESCRIPTION
**Pull request by mselwa**
Customizing Leuven-graphics for chapter 3 and using existing modules/toolchains in examples.

### CODE REVIEW

**Comment by boegel** [comment]
@mselwa this includes everything else too, which makes it hard to review...

**Comment by mselwa** [comment]
I am not sure what you mean by „everything else”. It is one tex file for chapter 3 and the changes are mainly including a number of ifleuven (but then also ifgent, ifantwerpen, ifbrussel) statements to show modules that we actually have. Perhaps looks starnge, but I do not want to include in our manual toolchains or software that we do not have.

**Comment by itkovian** [comment]
@mselwa If you look at the 'files' there are 22 files changed in this PR. If you look at the commits, there seem to be a bunch of commits that are unrelated to that one tex file.

File: intro-HPC/ch_preparing_the_environment.tex [hunk 1]
```diff
@@ -402,9 +484,43 @@ \subsection{Connect}
...
 Open the .bash\_profile or the .profile on your local machine with your
+\fi
+\ifbrussel
+Open the .bash\_profile or the .profile on your local machine with your
+\fi
+\ifgent
+Open the .bash\_profile or the .profile on your local machine with your
+\fi
+\ifleuven
+Open the .bashrc or the .profile on your local machine with your
```
Comment (JensTimmerman) [inline intro-HPC/ch_preparing_the_environment.tex]: is there any reason why .bash_profile doesn't work in leuven?
shouldn't we simply state all 3 for all institutes?

Comment (mselwa) [inline intro-HPC/ch_preparing_the_environment.tex]: It works. We just prefer our users to edit .bashrc - then we recognize those that modified anything...

Comment (JensTimmerman) [inline intro-HPC/ch_preparing_the_environment.tex]: I guess if you want to fix this, it should be specified in bashrc, but this issue is generally only seen in mac OS X, where .bash_profile is always executed, also on non login shells, so this always fixed it. bashrc is better thought

**Comment by JensTimmerman** [comment]
@mselwa when creating a new feature branch, you should first checkout the master branch again, so you branch from master, instead of from the feature branch you were on. Then the changes in that feature branch will not be included in the new branch. 

I have rebased your pr here on master and fixed my own remarks, and now it merges, please see https://github.com/hpcugent/vsc_user_docs/pull/121
and merge if you are happy with it.

File: intro-HPC/ch_preparing_the_environment.tex (hunk 1 above)
Comment (JensTimmerman) [inline intro-HPC/ch_preparing_the_environment.tex]: This line is talking about .bashrc on your local machine, so you can't see this, right?

**Comment by klust** [comment]
Mag, I've merged a pull request that Jens created from yours, so I think all changes in this pull request should be incorporated now. If this is the case, you can close this pull request.

**Pull request closed without merging**